Core.ApplicationServices/Organizations/OrganizationService.cs

```diff
@@ -224,7 +224,7 @@ public IQueryable<Organization> SearchAccessibleOrganizations(params IDomainQuer
             }
 
             var query = new IntersectionQuery<Organization>(domainQueries);
-            return _repository.GetAll().Transform(query.Apply);
+            return _repository.GetAll().OrderBy(prp => prp.Name).Transform(query.Apply);
```

Comment:
move the ordering to AuthorizeController::GetOrganizations

in that endpoint, take an optional query parameter orderBy=string, orderAsc=boolean (see the todos)